### ISSUE TITLE
Windows games in the 'other' category don't launch at all.

### ISSUE THREAD

**Issue by cosmopock** [Heroic-Games-Launcher/HeroicGamesLauncher#3567]
### Describe the bug

After the latest update of Heroic Launcher, my games stopped working, and I received a Windows Defender alert about Trojan:Win32/Phonzy.B!ml located in the file: C:\Users\XXX\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl.exe.
Every other store Epic , Amazon, GOG works.

### Add logs

```text
Start-Process : Missing an argument for parameter 'ArgumentList'. Specify a parameter of type 'System.String[]' and try
 again.
At line:1 char:114
============= End of log =============+ ... e\BendGame\Binaries\Win64\DaysGone.exe`"" -Wait -ArgumentList  -NoNew ...
+                                                     ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Start-Process], ParameterBindingException
    + FullyQualifiedErrorId : MissingArgument,Microsoft.PowerShell.Commands.StartProcessCommand
```

### Steps to reproduce

1. Go to 'Libary/Other games'
2. Click on 'run game '
3. See no error game don't lounch

### Expected behavior

the game should run

### Heroic Version

Latest Stable

### System Information

Windows 10 LTSC 2021

**Comment by arielj** [comment]
You can use the build from this PR https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3562 until a new release is done with that fix

**Comment by cosmopock** [comment]
THX for the reply but i'm to stupid for the solution sorry

**Comment by arielj** [comment]
You have to go here https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/actions/runs/7886383579?pr=3562 and scroll down. There's an `Artifacts` section and click on `win-portable`, this will download the portable version of Heroic for windows. You can then unzip it and run it from there until the next release.

**Comment by cosmopock** [comment]
Thank you games work but i'm getting this error when i try to add difrent games:
Gog
Błąd:
Error running command "GOGDL_CONFIG_PATH=C:\Users\ITX\AppData\Roaming\heroic\gogdlConfig powershell Start-Process "`"C:\Users\ITX\AppData\Local\Temp\2cJQX0hYCUOigIiVjWAetRQpG6f\resources\app.asar.unpacked\build\bin\win32\gogdl`"" -Wait -NoNewWindow -ArgumentList "`"--auth-config-path`"","`"C:\Users\ITX\AppData\Roaming\heroic\gog_store\auth.json`"","`"auth`""": Error: spawn powershell ENOENT
at Process.onexit (node:internal/child_process:283:19)
at onErrorNT (node:internal/child_process:476:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

**Comment by flavioislima** [comment]
are you using the Setup or the Portable version of heroic?

**Comment by Wilk-22** [comment]
I have the same issue, here are the logs to confirm that it's indeed a bug
```
Start-Process : Missing an argument for parameter 'ArgumentList'. Specify a parameter of type 'System.String[]' and 
try again.
At line:1 char:93
+ ... \C:\Games\Total War - Rome 2\Rome2.exe`"" -Wait -ArgumentList  -NoNew ...
============= End of log =============+                                                     ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Start-Process], ParameterBindingException
    + FullyQualifiedErrorId : MissingArgument,Microsoft.PowerShell.Commands.StartProcessCommand
``` 
```
Start-Process : Missing an argument for parameter 'ArgumentList'. Specify a parameter of type 'System.String[]' and 
try again.
At line:1 char:98
+ ... :\Games\Total War Saga - Troy\Troy.exe`"" -Wait -ArgumentList  -NoNew ...
============= End of log =============+                                                     ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Start-Process], ParameterBindingException
    + FullyQualifiedErrorId : MissingArgument,Microsoft.PowerShell.Commands.StartProcessCommand
```

**Comment by arielj** [comment]
I understand this was fixed in one of the latest relases (I think 2.14.0). Closing

**Comment by rafalohaki** [comment]
same issue
